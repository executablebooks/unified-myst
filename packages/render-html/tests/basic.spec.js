import { ProcessorHtml } from '../src/index.js'

describe('Parser', () => {
    test('parse', () => {
        const parser = new ProcessorHtml()
        const result = parser.toHtml('hallo world!')
        expect(result).toMatchSnapshot()
    })
})
