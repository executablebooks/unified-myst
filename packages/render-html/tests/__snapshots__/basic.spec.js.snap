// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Parser parse 1`] = `"<p>hallo world!</p>"`;
